core/modules/widgets/edit-text.js

```diff
@@ -212,17 +212,30 @@ EditTextWidget.prototype.updateEditorDomNode = function(text) {
 };
 
 /*
+Get the first parent element that has scrollbars or use the body as fallback.
+*/
+EditTextWidget.prototype.getScrollContainer = function(el) {
+	while(el.parentNode) {	
+		el = el.parentNode;
+		if(el.scrollTop) {
+			return el;
+		}
+	}
+	return document.body;
+};
+
+/*
 Fix the height of textareas to fit their content
 */
 EditTextWidget.prototype.fixHeight = function() {
-	var self = this,
-		domNode = this.domNodes[0];
+	var domNode = this.domNodes[0];
 	if(this.editAutoHeight && domNode && !domNode.isTiddlyWikiFakeDom && this.editTag === "textarea") {
 		// Resize the textarea to fit its content, preserving scroll position
-		var scrollPosition = $tw.utils.getScrollPosition(),
-			scrollTop = scrollPosition.y;
-		// Measure the specified minimum height
-		domNode.style.height = self.editMinHeight;
+		// Get the scroll container and register the current scroll position
+		var container = this.getScrollContainer(domNode),
+			scrollTop = container.scrollTop;
+                // Measure the specified minimum height
+		domNode.style.height = this.editMinHeight;
```

Comment:
@Jermolene that is an interesting way to allow the user to use calc() and percent units… I mean you assign the minHeight to the domNode just to measure it and the override it again with "auto", that seems strange on first sight – but considering, that the user may enter values that are not pixels, it is very clever to calculate the height that way.